### ISSUE TITLE
Common name exported incorrectly for xml indexing

### ISSUE THREAD

**Issue by AntonPetrov** [RNAcentral/rnacentral-webcode#75]
Some entries have just one letter, which is wrong.

Example:

`$ grep '"common_name">A<' *.xml`

```
xml4dbdumps__0__200000.xml:<field name="common_name">A</field>
xml4dbdumps__1200000__1400000.xml:<field name="common_name">A</field>
xml4dbdumps__1200000__1400000.xml:<field name="common_name">A</field>
xml4dbdumps__1200000__1400000.xml:<field name="common_name">A</field>
```

**Comment by blakesweeney** [comment]
I looked into this a bit in the database and found the following suspicious common names:

```csv
"'guitanii'",5
"-",2
"0",17
"0.1",4
"08",2
"09",2
"09.20.2007.11.2",2
"09.20.2007.13",2
"1",337
"1-3",1
"1.5",2
"1/59",10
"1/77",19
"10",10
"10.4.21",8
"10.5.11",3
"100.10",1
"100.12",1
"100.14",2
"100.15",1
"102a",110
"105a",121
"105zx",1
"106a",117
"107.2",1
"107.9",4
"10e",461
"11",10
"113zx",1
"114zx",1
"116zx",1
"117.20",1
"117.9",1
"117zx",1
"118zx",1
"11a",1
"11b",1
"12",24
"120zx",1
"121zx",1
"126.2",1
"127.12",1
"13",13
"130zx",1
"131.1",1
"1314",2
"1316",1
"1318",1
"133.2",1
"133.4",1
"136zx",1
"14",12
"143.1",1
"143.2",1
"143.3",1
"143.4",1
"143.5",1
"143.7",1
"143.8",1
"145.5",1
"146zx",1
"14b",79
"14d",8
"14zx",1
"15",8
"15a",1
"16",7
"169a",125
"16a",126
"16b",1
"16s",10
"17",7
"171a",93
"174a",109
"175a",121
"1788",1
"18",5
"18/77",2
"182a",127
"185a",106
"186a",106
"1899",42
"18a",1
"18zx",1
"19",5
"19/2",2
"1903",1
"1905)",12
"1951",23
"195a",123
"1976",2
"1993",96
"1994",1
"19a",1
"1AA2",2
"1a",2
"1aa2",1
"1b",1
"1c",1
"2 - 7",4
"2",196
"2-13",189
"2-f6",3
"2.1",2
"2.6",2
"2/59",4
"2/77",4
"20",10
"2000708",52
"2003",2
"2003a",1
"2003b",2
"2005",226
"2005-2",1
"2005b",3
"2006",92
"2006), non A.H Sm. & D.E. Stuntz",2
"2006b",2
"2007",52
"2007b",4
"2008",205
"2008a",1
"2008b",8
"2009",433
"2009b",8
"2010",2558
"2010a",1
"2010b",17
"2010c",1
"2011",1645
"2011b",24
"2011c",1
"2012",2
"2013",4
"2014",1
"2015",1
"202a",111
"20a",1
"20zx",1
"21",4
"21.9",1
"21/59",2
"22",4
"22-10",4
"23",1
"24",7
"24-pointed ladybird beetle",36
"25",2
"25a",114
"25zx",1
"26",3
"26.10",2
"26.9",1
"27",1
"27-11",12
"27zx",2
"28",3
"29",2
"2a",1
"2p1",1
"2s",2
"2zx",1
"3",92
"3-7155360",100
"3-8296502",107
"3-9175482",122
"3-9305343",111
"3-9341900",91
"3-9344314",97
"3.2",1
"3/59",4
"3/77",2
"30",4
"30.3",1
"31",1
"3108W2",2
"3108w2",1
"317a",82
"32",1
"34",1
"35",1
"358a",108
"36",1
"37",1
"37/77",2
"38zx",1
"39-zx",1
"3A-A3",2
"3a-a3",1
"3s",3
"3zx",1
"4",52
"4-0631051",110
"4-0792144",124
"4-0888028",101
"4-0907367",105
"4-0985554",113
"4-1021478",116
"4-1022806",90
"4-1034782",125
"4-1075578",113
"4-1119338",100
"4-1219782",109
"4-1273116",115
"4-2051087",108
"4-2100061",110
"4-2124971",117
"4-2153033",102
"4-2172526",97
"4-2173468",104
"4-2174936",317
"4-2209276",107
"4-2258986",122
"4-2262045",112
"4-2277670",117
"4-2298570",107
"4-2356805",99
"4-2356988",108
"4-2393174",104
"4-2428830",113
"4-2512823",108
"4-2538717",116
"4-2542528",101
"4-2593475",81
"4-2602156",110
"4-2605759",108
"4-2634716",121
"4-2646161",112
"4-2658593",107
"4-2661829",96
"4-2677849",123
"4-2679949",112
"4-2735946",92
"4-2736020",110
"4-2758776",108
"4-2773848",97
"4-2774682",95
"4-2837072",104
"4-2839709",121
"4-2850729",112
"4-2876612",98
"4-2876639",101
"4-2962667",120
"4-2963531",121
"4-2976528",99
"4-2977443",108
"4-3008371",123
"4-3026949",116
"4-3041891",91
"4-3042480",108
"4-3042928",121
"4-3042952",86
"4-3051617",104
"4-3054470",101
"4-3062198",128
"4-3062651",91
"4-3094677",100
"4-3112292",104
"4-3112802",108
"4-3113221",110
"4-3119890",114
"4-3126218",99
"4-3128229",120
"4-3129918",126
"4-3136817",104
"4-3163997",113
"4-3189163",117
"4-3189767",108
"4-3191362",119
"4-3191958",113
"4-3192490",100
"4-3192644",96
"4-3205932",126
"4-3206106",110
"4-3206505",124
"4-3220125",110
"4-3224287",92
"4-3248542",126
"4-3255514",119
"4-3258106",93
"4-3331423",113
"4-3343286",131
"4-3405044",86
"4-3405184",108
"4-3406229",94
"4-3418073",123
"4-3428664",98
"4-3447545",78
"4-3773988",98
"4-4449320",124
"4-4449924",113
"4-4451880",93
"4-4451937",118
"4-4459431",105
"4-4465989",107
"4-4465997",79
"4-4466217",117
"4-4466519",122
"4-4471660",108
"4-4500903",113
"4-4509048",99
"4-4517873",81
"4-4530470",112
"4-4638751",111
"4-4697553",120
"4-5627451",96
"4-5627982",117
"4-5636698",109
"4-5670322",114
"4-5672112",116
"4-5674999",109
"4-5675286",97
"4-5695937",104
"4-5703913",92
"4-5755573",111
"4-5799287",99
"4-5818141",106
"4-5851939",99
"4-5854636",122
"4-5860571",96
"4-5865042",118
"4-5876842",107
"4-5885604",114
"4-5893887",100
"4-5894387",119
"4-5904188",119
"4-5930790",122
"4-5934869",107
"4-5953081",110
"4-5970458",106
"4-5977630",113
"4-5985145",98
"4-5987253",92
"4-5995973",99
"4-6034225",114
"4-6066488",109
"4-6066704",90
"4-6073441",122
"4-6073565",121
"4-6074464",115
"4-6832164",106
"4-6851606",112
"4-6857191",113
"4-6876161",105
"4-6876862",107
"4-6877826",119
"4-6879578",105
"4-6879942",127
"4-6881831",111
"4-6906788",87
"4-6924867",113
"4-6942539",105
"4-6943160",112
"4-6977162",106
"4-6977960",100
"4-6978681",122
"4-6978754",112
"4-7026218",121
"4-7030436",87
"4-7036876",81
"4-7037740",108
"4-7039018",106
"4-7148410",96
"4-7154984",104
"4-7276001",108
"4-7276109",111
"4-7303629",112
"4-7315031",119
"4-7345399",126
"4-7449267",104
"4-7473087",95
"4-7535473",96
"4-7634056",98
"4-7649002",103
"4-7653042",92
"4-7787030",93
"4-8589585",84
"4.21",3
"40",2
"40l3",1
"40s1a",1
"40s1e",1
"43zx",1
"44",1
"44zx",1
"46",1
"47zx",1
"49zx",1
"4a",93
"4zx",1
"5",45
"5.21",7
"5.22",4
"5/59",2
"5/77",2
"506",1
"51",1
"532 C",1
"53zx",1
"5a",1
"5s",6
"5zx",1
"6",25
"6/59",2
"60",1
"60l1bb",1
"60l7",1
"60s3",1
"62.16",1
"63",1
"63a",91
"63zx",1
"64zx",1
"66a",108
"68.4",1
"68.8",1
"68a",112
"6e2s",1
"7",18
"72hr",6
"72zx",1
"75.3",1
"75.4",1
"76.8",1
"7e",1
"8",20
"81.19",1
"81.2",1
"81.3",1
"81.5",1
"81zx",1
"89.15",1
"9",26
"9-3",1
"94.1",1
"94.11",1
"967",1
"97.16",1
"97.3",1
"A",82
"A-i",1
"A-ii",1
"A-iii",1
"A1",15
"A14",4
"A2",6
"A4",4
"A52",4
"A6",4
"A83",4
"A91",4
"AC010",4
"AC021",4
"ARG",1
"Atkin's robber frog",38
"G",6
"Iran",33
"K",3
"Opah",26
"R",137
"agent of rickettsialpox",8
"alexanders",14
"blacksmith",47
"comet",5
"inconnu",24
"nyala",30
"sauger",29
"Ac010",2
"Ac021",2
"Ach5",77
"Ag",1
"Agrostis",2
"Aimara",1
"Alv2",1
"Amerizus",1
"Ames",61
"Apollo",1
```

this isn't all of them by far (I sorted and didn't get through all the A's), but I got tired of looking at these things.

**Comment by blakesweeney** [comment]
While there are still strange things indexed as `common_name` (i.e. http://test.rnacentral.org/search?q=common_name:%222%20-%207%22) this is an accurate representation of the data we are given. I think this will be fixed once we take care of #332. The issue is basically caused by organisms that have `(something)` in their name, which is parsed as the common name (because that is what the embl spec says it is).
